src/Common/Commands.Common/Commands.Common.csproj

```diff
@@ -157,14 +141,6 @@
     <Compile Include="AzurePowerShell.cs" />
     <Compile Include="ComputeCloudException.cs" />
     <Compile Include="Constants.cs" />
-    <Compile Include="Extensions\DSC\DscExtensionCmdletConstants.cs" />
```

Comment:
Both of these projects include commands.common, so this is not actually necessary - many commands put their resources in that resources file.